pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>sinonjs</artifactId>
-    <version>1.7.3-2-SNAPSHOT</version>
+    <version>1.17.2-1-SNAPSHOT</version>
```

Comment:
Usually we only do the `-1` thing for a revision / fix to the same upstream version.  So would you mind changing this to `1.17.2-SNAPSHOT` ?